src/orgSettings.ts

```diff
@@ -39,6 +39,11 @@ interface StaticFileSettings {
 export interface StaticFileProxySettings {
   useBaseplateHosting: boolean;
   host: string;
+  aws: {
```

Comment:
```suggestion
  aws?: {
```

Reply:
Oh cool, I didn't know that was a thing.